test/WasScreen.test.ts

```diff
@@ -0,0 +1,143 @@
+jest.mock('@react-native-async-storage/async-storage', () => ({
+  getItem: jest.fn(),
+}));
+
+import AsyncStorage from '@react-native-async-storage/async-storage';
+
+// Mock the function directly since we can't import it easily
+const WAS_BASE_URL = 'https://test-was-server.com';
+
+// Copy the function here for testing since we can't import it easily
```

Comment:
Hehe, see, this is a signal that it's being exported in the wrong place :)

Most business logic / reusable functions should go into the `app/lib/` folder or something similar, and live in a `.ts` file. Exactly for this kind of reason -- so that it's easier to import. Move it there.